src/test/java/org/assertj/core/api/atomic/longadder/LongAdderAssert_isNull_Test.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api.atomic.longadder;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.atomic.LongAdder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+
+class LongAdderAssert_isNull_Test {
```

Comment:
Any particular reason why you want to test `isNull()`? `LongAdderAssert` doesn't ovverride it from `AbstractAssert` so I'd probably skip it

Reply:
The `isNull` assertion or just the test?

Reply:
Just the test, the assertion should come from `AbstractAssert` and it is covered by `AbstractAssert_isNull_Test`